config/config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
```

Comment:
I like the idea of having a single file for configurations.

Reply:
Replacing constants file with literal values in `config.js` will make it hard to access default values as they are mixed with configs from `process.env`.  In general, those values can be needed if you want to check if a config is not following the default value or if resetting values to defaults is needed.

Let's either create a defaultConfig object or keep the constant.js

Reply:
Also, these configurations seem to be server configurations. Is there any reason to move them into the root directory? Usually, scoping the configs can be more secure to prevent leaking values to the client.